.changeset/real-toys-look.md

```diff
@@ -1,5 +1,9 @@
 ---
-"github-vscode-theme": minor
+"github-vscode-theme": major
```

Comment:
I was wondering about this.

From a code perspective, removing the Beta label feels more like a `patch`, but I guess from a marketing perspective `major` seems right. I guess with this theme we don't really follow Semver and do more a "consumer versions".

Reply:
I wasn't sure about this either. 🤔 I would say in this case because it's a consumer product we should use `major`.